lib/communityEvent/index.spec.js

```diff
@@ -0,0 +1 @@
+import '.'; // It doesn't crash.
```

Comment:
Could we not be checking the structure of the communityEvent is setup properly? I.e that displayLevel gets added to normal eventFields?